app/lib/actions/climbing/actions.ts

```diff
@@ -337,20 +338,34 @@ export async function deleteMember(
 
 // Fonction pour supprimer plusieurs membres de la base de données
 export async function deleteMembers(
-  ids: { id: string; imageUrl: string }[],
+  ids: string[],
 ): Promise<{ message: string }> {
-  if (ids.length === 0) {
-    return { message: 'Aucun membre à supprimer.' };
-  }
-
   try {
-    await Promise.all(
-      ids.map(({ id, imageUrl }) => deleteMember(id, imageUrl)),
-    );
+    if (ids.length === 0) {
+      return { message: 'Aucun membre à supprimer.' };
+    }
+    const placeholders = ids.map((_, index) => `$${index + 1}`).join(', ');
```

Comment:
Je ne comprends pas là...
On prends pas l'index mais l'id qu'il y a dans le ids nan?
```suggestion
    const placeholders = ids.join(', ');
```

Reply:
Without this method, the query isn't working => In the SQL query, you use const placeholders = ids.join(', ');. This will simply create a string containing the IDs separated by commas (for example, 1, 2, 3), which can be interpreted as literal values in SQL, but does not meet the expectations for placeholders if you are using a method that requires placeholders (like $1, $2, etc.).